src/switch_rtp.c

```diff
@@ -7673,7 +7758,7 @@ static void check_timeout(switch_rtp_t *rtp_session)
 		elapsed = (now - rtp_session->last_media) / 1000;
 	}
 
-	switch_log_printf(SWITCH_CHANNEL_SESSION_LOG(rtp_session->session), SWITCH_LOG_DEBUG10,
+	switch_log_printf(SWITCH_CHANNEL_SESSION_LOG(rtp_session->session), SWITCH_LOG_WARNING,
```

Comment:
These will pollute our b2bua logs. Imagine if we have 500 concurrent calls then we are gonna received a lot of this logs and will just ends up ignoring it. Will the info provided in line 7771 and 7775 would be enough to debug the media timeout?

Reply:
That's exactly why I changed logging level here, cause I want to see these in logs. This will only "pollute" logs if there is media timeout issue, in which case this issue should be fixed and these messages will not "pollute" logs anymore.

Reply:
No this will always call for validation. If you put your log under this statement
```
if (elapsed > rtp_session->media_timeout) {
}
```
Then this will only log when there is a timeout issue.

Reply:
There are other logs for that case. Reduced logging level for 7761 though.